docs/conf.py

```diff
@@ -61,7 +61,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'F5 OpenStack LBaaSv2 Driver'
+project = u'F5 OpenStack LBaaSv2 Plugin'
```

Comment:
Ok, as per discussion with @swormke, I'm going to remove 'plugin' so it's just 'LBaaSv2'.